src/stories/components/topbar/topbar.tsx

```diff
@@ -40,11 +40,11 @@ const AppBar = styled(MuiAppBar, {
 }));
 
 const TopBar = (props: TopBarProps) => {
-  return <AppBar position="absolute" open={props.open}>
+  return <AppBar position="absolute" open={props.open} style={{ backgroundColor: '#C4C4C4' }}>
     <Toolbar
       sx={{
         pr: '24px',
-        backgroundColor: 'white',
+        backgroundColor: '#C4C4C4',
```

Comment:
Let's leave the color as it was if possible. This isn't the final design anyways.